github-actions/bazel/setup/setup-ci-bazelrc.sh

```diff
@@ -0,0 +1,17 @@
+echo """
+# Print all the options that apply to the build.
+# This helps us diagnose which options override others
+# (e.g. /etc/bazel.bazelrc vs. tools/bazel.rc)
+build --announce_rc
+
+# Avoids re-downloading NodeJS/browsers all the time.
+build --repository_cache=$HOME/.cache/bazel_repo_cache
```

Comment:
This ends up finding the wrong location on windows if I remember correctly

Reply:
We were using a slash here as well, so it never really worked I think. Fixed by using a realpath